src/medida/timer.cc

```diff
@@ -245,20 +251,14 @@ void Timer::Impl::Clear() {
 void Timer::Impl::Update(std::chrono::nanoseconds duration) {
   auto count = duration.count();
   if (count >= 0) {
-    histogram_.Update(count);
+    histogram_.Update(count / duration_unit_nanos_);
```

Comment:
I didn't notice that `Histogram`'s `Update` function takes an integer. I thought it took doubles since functions like `min`, `max` and `sum` are all doubles. Will update it!

Reply:
Done! It's slightly annoying that we'll end up with an unused variable in `src/medida/timer.cc`, but I think it's okay?:

```
stats::Snapshot Timer::Impl::GetSnapshot(uint64_t divisor) const {
  // Timer should always use duration_unit_nanos_ as the divisor
  // since that is the only thing that makes sense.
  // However, Snapshot class inherits GetSnapshot from SamplingInterface
  // which forces GetSnapshot to have an argument.
  // Therefore, we will ignore the divisor argument, and pass duration_unit_nanos_.
  return histogram_.GetSnapshot(duration_unit_nanos_);
}
```

Reply:
(This was fixed differently -- resolved)